resources/views/pages/edit_post.blade.php

```diff
@@ -0,0 +1,116 @@
+                            <option value="{{ $post->place->province->id }}" selected>{{ $post->place->province->prov_name }}</option>
+                            @foreach ($provinces as $prov)
+                                <option value="{{ $prov->id }}">{{ $prov->prov_name }}</option>
+                            @endforeach
+                        </select>
+                    </label>
+            </div>
+            <div class="form-group">
+                <textarea 
+                    name="intro" 
+                    placeholder="{{ trans('post.intro') }}" 
+                    rows="3" 
+                    class="form-control"
+                    >{{ $post->intro }}</textarea>
+            </div>
+            <div class="form-group">
+                <h4>{{ trans('post.para') }}</h4>
+            </div>
+
+            <div class="form-group">
+                <!-- Upload image input-->
+                <div class="input-group mb-3 px-2 py-2 rounded-pill bg-white shadow-sm">
+                    <input id="upload" type="file" class="form-control border-0" name="images[]">
+                    <label id="upload-label" class="font-weight-light text-muted">{{ trans('post.choose_img') }}</label>
+                    <div class="input-group-append">
+                        <label for="upload" class="btn btn-light m-0 rounded-pill px-4"> 
+                            <i class="fa fa-cloud-upload-alt mr-2 text-muted"></i>
+                            <small class="text-uppercase font-weight-bold text-muted">{{ trans('post.choose_img') }}</small></label>
+                    </div>
+                </div>
+
+                <!-- Uploaded image area-->
+                <p class="font-italic text-center">{{ trans('post.uploaded') }}</p>
+                <div class="image-area mt-4"><img id="imageResult" src="{{ asset($post->images()->latest()->first()->url) }}" alt="" class="img-fluid rounded shadow-sm mx-auto d-block"></div>
+
+            </div>
+
+            <div class="form-group">
+                <textarea 
+                    name="content" 
+                    placeholder="{{ trans('post.content') }}" 
+                    rows="5" 
+                    class="form-control"
+                    >{{ $post->content }}</textarea>
+            </div>
+            <div class="form-group text-center">
+                <button type="submit" class="btn boxed-btn4">
+                    {{ trans('post.create_button') }}
+                </button>
+            </div>
+        </div>
+    </div>
+</form>
+@endsection
+
+@section('script')
+    <script src="{{ asset('bower_components/review_travel/js_travel/post.js') }}"></script>
+    <script src="{{ asset('bower_components/select2/dist/js/select2.min.js') }}"></script>
+    <script src="{{ asset('bower_components/review_travel/js_travel/select.js') }}"></script>
+    <script src="{{ asset('js/all.js') }}"></script>
```

Comment:
nếu đã mix thì dùng như này nhé chị
```suggestion
    <script src="{{ mix('js/all.js') }}"></script>
```

Reply:
okie, cam on hoang nhe :3